autometrics-macros/src/lib.rs

```diff
@@ -57,8 +57,12 @@ fn instrument_function(args: &AutometricsArgs, item: ItemFn) -> Result<TokenStre
     let prometheus_url =
         env::var("PROMETHEUS_URL").unwrap_or_else(|_| DEFAULT_PROMETHEUS_URL.to_string());
 
-    // Build the documentation we'll add to the function's RustDocs
-    let metrics_docs = create_metrics_docs(&prometheus_url, &function_name, args.track_concurrency);
+    // Build the documentation we'll add to the function's RustDocs, unless it is disabled by the environment variable
+    let metrics_docs = if option_env!("AUTOMETRICS_DISABLE_DOCS").is_some() {
```

Comment:
`option_env!` works at compile-time. Is that really what we want? I see `PROMETHEUS_URL` uses the regular `env::var()` function, which I think I would've expected here too.

Reply:
TBH, I'm not sure it makes a difference because the macros are only run at compile time. But I can change it anyway

Reply:
Yeah, I also found it hard to reason about what it means for something to be done at compile-time for a macro. Maybe avoiding the confusion is reason enough :)